addons/hr_expense_stripe/models/hr_employee.py

```diff
@@ -1,170 +1,22 @@
-from odoo import _, api, fields, models
-from odoo.exceptions import UserError, ValidationError
-
+from odoo import fields, models
+from odoo.addons.hr_expense_stripe.utils import make_request_stripe_proxy
 
 class HrEmployee(models.Model):
     _name = 'hr.employee'
-    _inherit = ['hr.employee', 'stripe.issuing']
+    _inherit = ['hr.employee']
 
-    stripe_id = fields.Char(string='Stripe ID', readonly=True, copy=False, index='trigram', groups="hr.group_hr_user")
-    private_first_name = fields.Char(string='First Name', compute='_compute_from_name', store=True, groups="hr.group_hr_user")
-    private_last_name = fields.Char(string='Last Name', compute='_compute_from_name', store=True, groups="hr.group_hr_user")
+    stripe_cardholder_id = fields.Char(string='Stripe ID', readonly=True, copy=False, index='trigram', groups="hr.group_hr_user")
```

Comment:
Why renaming it?
My issue with the _id was that it was hiding the fact it was actually a Char
By calling them all stripe_id it allowed some kind of consistency, here it's harder to understand in comparison to the stripe_credit_card_ids which is indeed a recordset